crates/control/src/referee_position_provider.rs

```diff
@@ -0,0 +1,64 @@
+use color_eyre::Result;
+use context_attribute::context;
+use coordinate_systems::{Field, Ground};
+use framework::MainOutput;
+use linear_algebra::{point, Isometry2, Point2};
+use serde::{Deserialize, Serialize};
+use types::{field_dimensions::FieldDimensions, world_state::WorldState};
+
+#[derive(Deserialize, Serialize)]
+pub struct RefereePositionProvider {}
+
+#[context]
+pub struct CreationContext {}
+
+#[context]
+pub struct CycleContext {
+    ground_to_field: Input<Option<Isometry2<Ground, Field>>, "ground_to_field?">,
+    world_state: Input<WorldState, "world_state">,
+
+    field_dimensions: Parameter<FieldDimensions, "field_dimensions">,
+    normed_expected_referee_position:
+        Parameter<Point2<Ground>, "detection.detection_top.normed_expected_referee_position">,
+}
+
+#[context]
+#[derive(Default)]
+pub struct MainOutputs {
+    pub expected_referee_position: MainOutput<Option<Point2<Ground>>>,
+}
+
+impl RefereePositionProvider {
+    pub fn new(_context: CreationContext) -> Result<Self> {
+        Ok(Self {})
+    }
+
+    pub fn cycle(&mut self, context: CycleContext) -> Result<MainOutputs> {
+        let Some(ground_to_field) = context.ground_to_field else {
+            return Ok(MainOutputs {
+                expected_referee_position: None.into(),
+            });
+        };
+        let mut normed_expected_referee_position = *context.normed_expected_referee_position;
```

Comment:
Because of the scaled down field in SMD. It has to be scaled to the correct field dimensions, otherwise at SMD the target for the `look_at` would be off. 
But I guess just removing the parameter entirely and calculate the position in the node/function entirely will circumvent this anyway.

Reply:
The parameter is now removed and the entire computation is now done in `referee_position_provider.rs`.